node-notes-cli/app.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const data = require('./data.json');
+
+// read a note
+if (process.argv[2] === 'read') {
+  fs.readFile('data.json', 'utf8', (err, data) => {
+    if (err) throw err;
+    const jsonObj = JSON.parse(data, null, 2);
+    for (const property in jsonObj.notes) {
+      console.log(`${property}: ${jsonObj.notes[property]}`);
+    }
+  });
```

Comment:
Since you required your data model at the top of this file, you can skip this `fs.readFile` and use the required data instead.